iothub/service/samples/getting started/ServiceClientSample/Program.cs

```diff
@@ -42,16 +42,21 @@ public static async Task<int> Main(string[] args)
                     // The SDK logs are written at Trace level. Set this to LogLevel.Trace to get ALL logs.
                     MinLogLevel = LogLevel.Debug,
                 });
-            var logger = loggerFactory.CreateLogger<Program>();
+            ILogger<Program> logger = loggerFactory.CreateLogger<Program>();
 
             // Instantiating this seems to do all we need for outputting SDK events to our console log.
             using var sdkLog = new ConsoleEventListener(SdkEventProviderPrefix, logger);
 
-            var runningTime = parameters.ApplicationRunningTime != null
+            TimeSpan runningTime = parameters.ApplicationRunningTime != null
                 ? TimeSpan.FromSeconds((double)parameters.ApplicationRunningTime)
                 : Timeout.InfiniteTimeSpan;
 
-            var sample = new ServiceClientSample(parameters.IoTHubConnectionString, parameters.TransportType, parameters.DeviceId, logger);
+            var serviceClientOptions = new IotHubServiceClientOptions
+            {
+                Protocol = IotHubTransportProtocol.Tcp
```

Comment:
You're hard coding this. You need to add `IotHubTransportProtocol` as a parameter.